src/main/groovy/org/gradlefx/tasks/AirPackage.groovy

```diff
@@ -65,6 +69,32 @@ class AirPackage extends DefaultTask {
         showAntOutput ant.properties[ANT_OUTPUT_PROPERTY]
     }
 
+    def initInputOutputFiles() {
+        if (project.type == FlexType.air) {
+            if (flexConvention.air.keystore) {
+                def keystore = project.file(flexConvention.air.keystore)
+                if (keystore.exists()) {
+                    inputs.files keystore.absolutePath
```

Comment:
I found this works, for I had logging statements at the end of this closure to iterate through the inputs and outputs to display them, and each file declared in this manner was displayed.

Note that no assignment (=) is used.